test/support/fixtures.ex

```diff
@@ -93,6 +93,22 @@ defmodule Apr.Fixtures do
     }
   end
 
+  def stripe_account_inactive_error_event() do
+    %{
+      "object" => %{
+        "id" => "Errors::FailedTransactionError",
+        "display" => "Errors::FailedTransactionError"
```

Comment:
#non-blocking. We can check if this is accurate. I'm guessing this is just the class name which is `FailedTransactionError`.